lighthouse-treemap/app/src/main.js

```diff
@@ -307,11 +312,106 @@ class TreemapViewer {
     };
   }
 
+  createTable() {
+    const tableEl = TreemapUtil.find('.lh-table');
+    tableEl.innerHTML = '';
+
+    /** @type {Array<{name: string, resourceBytes: number, unusedBytes?: number}>} */
+    const data = [];
+    TreemapUtil.walk(this.currentTreemapRoot, (node, path) => {
+      if (node.children) return;
+
+      // Elide the first path component, which is common to all nodes.
+      let name;
+      if (path[0] === this.currentTreemapRoot.name) {
+        name = path.slice(1).join('/');
+      } else {
+        name = path.join('/');
+      }
+
+      // Elide the document URL.
+      if (name.startsWith(this.currentTreemapRoot.name)) {
+        name = name.replace(this.currentTreemapRoot.name, '//');
+      }
+
+      data.push({
+        name,
+        resourceBytes: node.resourceBytes,
+        unusedBytes: node.unusedBytes,
+      });
+    });
+
+    /** @param {Tabulator.CellComponent} cell */
+    const makeCoverageTooltip = (cell) => {
+      /** @type {typeof data[number]} */
+      const dataRow = cell.getRow().getData();
+      if (!dataRow.unusedBytes) return '';
+
+      const percent = Math.floor(100 * dataRow.unusedBytes / dataRow.resourceBytes);
+      return `${percent}% bytes unused`;
+    };
+
+    const gridEl = document.createElement('div');
+    tableEl.append(gridEl);
+
+    const maxSize = this.currentTreemapRoot.resourceBytes;
+    this.table = new Tabulator(gridEl, {
+      data,
+      height: '100%',
+      layout: 'fitColumns',
+      tooltips: true,
+      addRowPos: 'top',
+      history: true,
+      resizableColumns: true,
+      initialSort: [
+        {column: 'resourceBytes', dir: 'desc'},
+      ],
+      columns: [
+        {title: 'Name', field: 'name', widthGrow: 5},
+        {title: 'Size', field: 'resourceBytes', formatter: cell => {
+          const value = cell.getValue();
+          return TreemapUtil.formatBytes(value);
+        }},
+        // eslint-disable-next-line max-len
+        {title: 'Unused', field: 'unusedBytes', widthGrow: 1, sorterParams: {alignEmptyValues: 'bottom'}, formatter: cell => {
+          const value = cell.getValue();
+          if (value === undefined) return '';
+          return TreemapUtil.formatBytes(value);
+        }},
+        // eslint-disable-next-line max-len
+        {title: 'Coverage', field: 'resourceBytes', widthGrow: 3, tooltip: makeCoverageTooltip, formatter: cell => {
```

Comment:
Should this be sorted by percentage so it doesn't give the same order as sorting the "Unused" column?